main/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ include(${GETTEXT_USE_FILE})
 gettext_compile(${GETTEXT_PACKAGE} SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/po TARGET_NAME ${GETTEXT_PACKAGE}-translations)
 
 find_packages(MAIN_PACKAGES REQUIRED
+    Gst
```

Comment:
Oh offcourse that stuff. Sorry I did not look at that commit when asking about the gst dep above and I forgot about it. Yeah I will redo that. I also realize that I should have explained in the commit msg what I was doing there. There was a bug because Gst.init wasnt called yet or something. It gets called from the RTP plugin.